src/toil/wdl/wdltoil.py

```diff
@@ -89,6 +98,13 @@
 
 logger = logging.getLogger(__name__)
 
+# WDL options to pass into the WDL jobs and standard libraries
+#   task_path: Dotted WDL name of the part of the workflow this library is working for.
+#   execution_dir: Directory to use as the working directory for workflow code.
+#   container: The type of container to use when executing a WDL task. Carries through the value of the commandline --container option
+WDL_Context = TypedDict('WDL_Context', {"execution_dir": NotRequired[str], "container": NotRequired[str],
```

Comment:
I wouldn't use an `_` in the name here; it's a different convention than all the other types.